Makefile.common.in

```diff
@@ -66,7 +66,7 @@ else
   ocamlopt_cmd = $(FLEXLINK_ENV) $(ocamlopt)
 endif
 
-OPTCOMPFLAGS=
```

Comment:
What is the impact of this change ? Is it possible to document whether it's a temporary fix until we get a better inliner, or a good setting for other reasons ?

Reply:
Yeah, I think once the inliner has been sorted this can go away.  I'll add a note.